src/js/pages/system/OverviewDetailTab.js

```diff
@@ -179,6 +188,101 @@ class OverviewDetailTab extends mixin(StoreMixin) {
     return publicIP;
   }
 
+  /**
+   * Get user if existent otherwise
+   * return null
+   *
+   * @returns {Component|Null} User string or null
+   *
+   * @memberOf OverviewDetailTab
+   */
+  getMesosBuildUser() {
+    const mesosConfig = MesosStateStore.get("lastMesosState");
```

Comment:
@nLight thanks for the reviews.. quick question though, why would you rather have this being passed?

Having the method `getMesosBuildUser()` to be the only to know about the `build_user` makes it easy to understand and follow the path if needed, we are not unit testing here but this would also help make better/more precise unit tests. (we wont gain any performance improvement here if passed instead since the data is store in a obj and "always available" for retrieving)

The method `getMesosDetails()` don't need to have logic for this particular data, it only need the data to be displayed. Another point is that the "clean code" patterns suggest not passing arguments to methods/funcs when avoidable.

All that being said if you have strong feelings against it I can move the logic to `getMesosDetails()`